doc/radosgw/config-ref.rst

```diff
@@ -610,12 +610,9 @@ Swift Settings
               Those containers cannot be versioned by the S3 object versioning
               mechanism.
 
-	      The ``X-History-Location`` attribute, also understood by
-	      OpenStack Swift for handling ``DELETE`` operations
-	      `slightly differently
-	      <https://docs.openstack.org/swift/latest/overview_object_versioning.html>`_
-	      from ``X-Versions-Location``, is currently not
-	      supported.
+	      A slightly different attribute, ``X-History-Location``, which is also understood by 
+              `OpenStack Swift <https://docs.openstack.org/swift/latest/api/object_versioning.html>`
```

Comment:
@falcon78921 you need to add an underscore after the backquote .

Reply:
see http://docs.ceph.com/ceph-prs/28774/radosgw/config-ref/

Reply:
@tchaikov Oops, missed that. I added the backquote and tested on my end. Should be good to go :)